File: 2533-bitwise-xor-of-all-pairings/2533-bitwise-xor-of-all-pairings.py
class Solution(object):
    def xorAllNums(self, nums1, nums2):
        res = 0
        if len(nums2)%2==1:
            for i in nums1:
                res =res^ i
        if len(nums1)%2==1:
            for i in nums2:
                res=res ^ i
        return res

File: 2533-bitwise-xor-of-all-pairings/2533-bitwise-xor-of-all-pairings.java
class Solution {
    public int xorAllNums(int[] nums1, int[] nums2) {
        int res = 0;
        if(nums2.length%2==1)
        {
            for(int i:nums1)
                res =res^ i;
        }
        if(nums1.length%2==1)
        {
            for(int i:nums2)
                res=res ^ i;
        }
        return res;
    }
}